pay-api/src/pay_api/models/payment.py

```diff
@@ -289,7 +289,7 @@ def generate_base_transaction_query(cls):
                 InvoiceReference.reference_number,
                 InvoiceReference.status_code,
             )
-            .outerjoin(PaymentAccount, Invoice.payment_account_id == PaymentAccount.id)
+            .join(PaymentAccount, Invoice.payment_account_id == PaymentAccount.id)
```

Comment:
This large query is our main query, tried to narrow down our count and subqueries so they come up a bit more simple